### DISCUSSION TITLE
Use of the Extensible Business Reporting Language (XBRL) For Machine Readable formats

### DISCUSSION THREAD

**Discussion by jstclair2019** [CMSgov/price-transparency-guide#4] [Q&A] [answered]
While the guide proposes non-proprietary, open formats for the required machine readable file, CMS has not mandated  specifically what syntax should be used or the specific semantics of what is actually provided.  At present, this appears to result in over 6000 hospitals posting files without a mandatory syntactic or semantic framework. It recommend that CMS adopt a semantic, financial reporting model and mandate the use of XBRL.

XBRL is the international standard for digital reporting of financial, performance, risk and compliance information, although it is also used for many other types of reporting. The open XBRL specifications are freely licensed to anyone seeking to use the standard. XBRL provides a way to:
- define unambiguous, reusable definitions
- report individual facts against those definitions
- where necessary (and permitted) extend those definitions to take account of unique reporting ideas or aggregations
- test the resulting report against the constraints set out in the definitions
- file or publish the finished report
- consume entire reports or individual facts as needed

Worldwide, data standards are gaining traction for reporting by public and private companies, as well as government entities. As noted by XBRL International, , more than 150 digital reporting mandates exist across more than 70 countries, requiring digital reporting using XBRL by more than 10 million entities, including private and public companies, and governments. In the U.S.,
entities reporting to the Securities and Exchange Commission (SEC), the Federal Deposit Insurance Corporation (FDIC), and the Federal Energy Regulatory Commission (FERC) are all required to submit data in XBRL format.

A preliminary working prototype is available for review here: http://xbrlsite.azurewebsites.net/2021/cma/

**Comment by shaselton-usds** [comment]
This isn't really an issue. Converting this to a discussion.

**Reply by jstclair2019** [reply]
Sounds good, @shaselton - issues generally lead to discussions. :)

**Comment by shaselton-usds** [comment, marked as answer]
Mandating a format, such as XBRL, goes against the intent of the rule in providing flexibility with implementation. By providing a standard schema, the ability to port from one format to the next is a much lighter lift than requiring industry to all conform to a single format.

**Reply by jstclair2019** [reply]
I'd point to the work done in SEC and FDIC in using XBRL for machine readable reports. I'm also not sure what the intent to "port to one format to the next" may be, as the current level of adoption doesn't provide either a standardized ontology for consumers, states, or CMS
